### PR TITLE
fix #77484 futurism.com

### LINKED ISSUE #77484: futurism.com [Ad Text String]
https://futurism.com/neoscope/scientists-cancer-drug-without-animal-tests

Ad text strings seen at various places. Checked with AG extension for Chrome.

https://prnt.sc/10l4hai

### PR DESCRIPTION
**Pull request by wout-codes**
#77484

### CODE REVIEW

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -258,6 +258,7 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 porn88.org###ad-player
+futurism.com##.hidden
```
Comment (Sergey-Lyapin) [inline EnglishFilter/sections/specific.txt:261]: blocks some useful elements as well

File: SocialFilter/sections/specific.txt [hunk 2]
```diff
@@ -1133,7 +1133,7 @@ newyorker.com##.cne-video-buttons
 denofgeek.com##.social-links__items
 denofgeek.com##.social-links__items + span.sep
 goldapple.ru##.pdp__share
-koronawirus.lol,kapwing.com,stopgame.ru,dailywire.com,donald.pl,speed.cloudflare.com,hackernoon.com##.react-share__ShareButton
+futurism.com,koronawirus.lol,kapwing.com,stopgame.ru,dailywire.com,donald.pl,speed.cloudflare.com,hackernoon.com##.react-share__ShareButton
```
Comment (Sergey-Lyapin) [inline SocialFilter/sections/specific.txt:1136]: We need to hide "SHARE THIS ARTICLE" inscription too.
In both cases it is better to use extended CSS.

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/specific.txt (hunk 1 above)
Comment (Alex-302) [reply EnglishFilter/sections/specific.txt:261]: bette `futurism.com##div[class="pb-2 text-center"] > h4`

File: SocialFilter/sections/specific.txt (hunk 2 above)
Comment (wout-codes) [reply SocialFilter/sections/specific.txt:1136]: any recommendations?

Comment (Sergey-Lyapin) [reply SocialFilter/sections/specific.txt:1136]: `futurism.com#?#main > div > div.text-center:has(> div.flex > button.react-share__ShareButton)`

and for the newsletter block:
`futurism.com#?##__next > div.antialiased > div > div.text-center:has(> div.mx-auto > div:contains(newsletter))
`

File: AnnoyancesFilter/sections/css_extended.txt [hunk 3]
```diff
@@ -11,6 +11,8 @@ amazon.de#?#.webPlayerElement .atvwebplayersdk-player-container div:not([class])
 overclockers.ru#?#.main-wrap > div.grid > div.container-block:has(> div > div > div.an-block-desc)
 ! https://github.com/AdguardTeam/AdguardFilters/issues/21282#issuecomment-441297983
 reddit.com#?#div[class][style="max-width:100%"] > div > div[class]:first-child:has(> button+h1:contains(Personalize your Reddit))
+https://github.com/AdguardTeam/AdguardFilters/issues/77484
+futurism.com#?##__next > div.antialiased > div > div.text-center:has(> div.mx-auto > div:contains(newsletter))
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/css_extended.txt:15]: to the main newsletter block please

Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/css_extended.txt:15]: under this comment
`!# Subscriptions/newsletters (not popups)`

**Pull request merged**